src/sass/_permission_confirm_modal.scss

```diff
@@ -0,0 +1,46 @@
+.permission-confirm-modal {
+  .confirm-table {
+    max-height: min(500px, 30vh);
```

Comment:
This seems rather small, why not use the fully available space? Though we need to encounter also for the "you" notification that might be visible below the table.

![image](https://github.com/canonical/lxd-ui/assets/1155472/59616ff7-a0db-46c0-b59f-fbde43734420)

Reply:
For smaller screens, if the max height for the table is too high then it causes double scrollbar issue, currently everything is dealt with using CSS, I guess I will write some JS to make it work.

Reply:
We could maybe use a scrollable container wrap the GroupsOrIdentityChangesTable?

Reply:
The `ScrollableContainer` component requires ids for elements below the child to work. Since this table is in a modal, I can't really assign an id to the modal footer. Maybe we can change the implementation of the ScrollableContainer to take in a list of selectors instead of ids :thinking:

Reply:
> ~We could maybe use a scrollable container wrap the GroupsOrIdentityChangesTable?~
> 
> Edit: We want the notification always be visible, maybe wrap the div with the className="confirm-table" with a scrollable container?

Sure that's what I meant, but the `ScrollableContainer` needs to be adjusted a bit since when calculating the offset to adjust the table in the modal, I also need the height of the modal footer in addition to the notification. Because the modal is a react component, I can't specify an id on the footer which is needed by the `ScrollableContainer` via the `belowIds` prop. It is working now, maybe we can have a quick chat about the approach on Monday so we are on the same page.

Reply:
![Screenshot from 2024-04-27 12-33-46](https://github.com/canonical/lxd-ui/assets/151511568/26526857-da68-4071-95bb-323bf53f626a)

Reply:
I think now it always takes the full height, which seems too big if there are few changes on a very large screen. I think it should shrink down and grow in height responsively

Reply:
Had to add some custom logic in the `GroupOrIdentityChangesTable` component. `ScrollableContainer` or `ScrollableTable` won't work 100% in this case because the offset is calculated based on `100vh`, which causes the above issue.

Reply:
looks good now :)